packages/desktop/views/dashboard/send-flow/SendFlowRouterView.svelte

```diff
@@ -15,10 +15,10 @@
 <send-flow-router>
     {#if $sendFlowRoute === SendFlowRoute.SelectToken}
         <SelectTokenView />
-    {:else if $sendFlowRoute === SendFlowRoute.InputTokenAmount}
-        <InputTokenAmountView />
     {:else if $sendFlowRoute === SendFlowRoute.InputDestination}
         <InputDestinationView />
+    {:else if $sendFlowRoute === SendFlowRoute.SelectAmount}
```

Comment:
this should be before the destination in the flow

Reply:
I think you forgot to move it before the destination.